include/opendht/dht_proxy_server.h

```diff
@@ -237,6 +238,9 @@ class OPENDHT_PUBLIC DhtProxyServer
     std::thread schedulerThread_;
 
     Sp<Scheduler::Job> printStatsJob_;
+    Sp<Scheduler::Job> getNodeStatsJob_;
+    mutable NodeStats ipv4Stats_ {};
```

Comment:
shouldn't be mutable

Reply:
I do this because I use this in getNodeInfo() which is const.

Reply:
IMO there really is no reason to update from getNodeInfo() (the goal of this patch is to put the load on a separate job).

Reply:
the only case I see is that we set up a dhtproxyserver and we directly try to do a listen on it. (this is the case in `dhtproxytester.cpp`)

I can just disable unit tests on the proxy till I found a better solution to do that.